test/system/roles/public_health/dashboard/export_verifier.rb

```diff
@@ -391,6 +412,41 @@ def validate_custom_export_assessmemts(patients, data, settings, export_file)
     end
   end
 
+  def validate_custom_export_vaccines(patients, data, _settings, export_file)
+    vaccines = vaccines_by_patient_ids(patients.pluck(:id))
+    vaccines_sheet = export_file.sheet('Vaccinations')
+    assert_equal(vaccines.size, vaccines_sheet.last_row - 1, 'Number of vaccines in Vaccinations List')
+
+    checked = data.dig(:vaccines, :checked)
+
+    # Validate vaccine headers
+    headers = checked.map { |field| ImportExport::VACCINE_FIELD_NAMES[field] }
+    headers.each_with_index do |header, col|
+      assert_equal(header, vaccines_sheet.cell(1, col + 1), "For header: #{header} in Vaccines")
+    end
+
+    # Validate vaccine cells
+    vaccine_row = 0
+    patients.find_each do |patient|
+      patient.vaccines.find_each do |vaccine|
+        # Get basic field data that is checked
+        vaccine_summary_arr = checked.map do |field|
+          if %i[user_defined_id_statelocal user_defined_id_cdc user_defined_id_nndss].include?(field)
```

Comment:
can we replace this array with `PATIENT_FIELD_TYPES[:alternative_identifiers]` since it's already defined there? (and can you do the same for `validate_custom_export_assessments` as well)?